src/scripts/main.js

```diff
@@ -18,10 +18,29 @@ const food = {
   },
 };
 
+const newTree = document.createElement('div');
+
+newTree.setAttribute('id', 'tree');
+
+document.body.prepend(newTree);
+
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  element.append(list);
+
+  for (const key in data) {
+    const item = document.createElement('li');
+
+    item.textContent = key;
+    list.append(item);
+
+    if (Object.keys(data[key]).length > 0) {
```

Comment:
The check for `Object.keys(data[key]).length > 0` is correct for determining if there are nested objects. This ensures that only non-empty objects are passed recursively.